src/common/components/vanilla-modules/icons/index.js

```diff
@@ -0,0 +1,202 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import styles from '../../../style/vanilla/css/icons.css';
+
+// Creates icon style class name
+const iconStyle = (icon = '') => {
+  icon = icon ? '--' + icon : '';
```

Comment:
This is a modifier.. classes in the framework all start with `p-icon--error` for the error icon and sizes or colour are e.g `p-icon--large` so for the classes to work I have applied this. Also the class `p-icon` can be used on a image say an SVG so it'll work the same way if required.